.github/workflows/labeler.yml

```diff
@@ -0,0 +1,14 @@
+name: "Pull Request Labeler"
+on:
+  - pull_request_target
+
+jobs:
+  triage:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@v4
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Does this token need to be created? I don't have the permission to do so.

Reply:
No, it's an [auto-generated token](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) for each process.